app/views/pages/home.html.erb

```diff
@@ -14,7 +14,7 @@
   </div>
 </div>
 <div class="container my-5">
-  <h1>ChompFood Promos in <strong style="color: #00b14f">Singapore</strong></h1>
+  <div class="pb-5"><h1>ChompFood Promos in <strong style="color: #00b14f">Singapore</strong></h1></div>
```

Comment:
can you make the div in its own line?
like 
<div>
  <h1>...</h1>
</div>

Reply:
You mean something like this? 

![Screen Shot 2021-09-19 at 2 59 20 am](https://user-images.githubusercontent.com/84091703/133905744-8cd8d73c-d28a-4f97-b3d7-45432f3e81f0.png)